openpower/package/sb-signing-utils/sb-signing-utils.mk

```diff
@@ -0,0 +1,35 @@
+################################################################################
+#
+#  sb-signing-utils
+#
+################################################################################
+
+SB_SIGNING_UTILS_SITE ?= $(call github,hellerda,sb-signing-utils,$(SB_SIGNING_UTILS_VERSION))
+SB_SIGNING_UTILS_SITE_VERSION = master
```

Comment:
Should rather be a tagged version where we can just pull the source tarball (from make-dist)